runtime/westend/src/lib.rs

```diff
@@ -1686,6 +1687,8 @@ sp_api::impl_runtime_apis! {
 				hex_literal::hex!("63f78c98723ddc9073523ef3beefda0ca95dac46c07a40d91506e7637ec4ba57").to_vec().into(),
 				// Configuration ActiveConfig
 				hex_literal::hex!("06de3d8a54d27e44a9d5ce189618f22db4b49d95320d9021994c850f25b8e385").to_vec().into(),
+				// The transactional storage limit.
+				hex_literal::hex!("3a7472616e73616374696f6e5f6c6576656c3a").to_vec().into(),
```

Comment:
Should be added to polakdot as well?

Reply:
But why then add it here? :P You also added it for Westend.

Reply:
yeah shouldn't matter if it is duplicated here after adding it to the benchmark layer.

all of this points that we should just have a better overall story for whitelisted storage when the time comes.

Reply:
Fixed here https://github.com/paritytech/substrate/pull/11394